src/main/java/io/github/swagger2markup/internal/document/PathsDocument.java

```diff
@@ -121,7 +116,7 @@ private void buildsPathsSection(MarkupDocBuilder markupDocBuilder, Map<String, P
         List<PathOperation> pathOperations = PathUtils.toPathOperationsList(paths, getBasePath(), config.getOperationOrdering());
         if (CollectionUtils.isNotEmpty(pathOperations)) {
             if (config.getPathsGroupedBy() == GroupBy.AS_IS) {
-                pathOperations.forEach(operation -> buildOperation(markupDocBuilder, operation));
+                pathOperations.forEach(operation -> buildOperation(markupDocBuilder, operation, config));
```

Comment:
You don't have to add config as a parameter. Documents and Components extend MarkupComponent